dependencies/graknlabs/dependencies.bzl

```diff
@@ -56,7 +56,8 @@ def graknlabs_console():
     git_repository(
         name = "graknlabs_console",
         remote = "https://github.com/graknlabs/console",
-        tag = "1.0.3",
+        # tag = "1.0.3",
```

Comment:
why are we only commenting out the tag reference and not actually removing it, @flyingsilverfin ?

Reply:
Errr can do, mostly as a reminder to revert it back to a tag when we release console...?

Reply:
That's already part of a `release-validation` check. No java repo can be released without depending on a `tag`, otherwise, release validation will fail. So we can remove the comment as it's part of natural workflow now.

Reply:
cool